src/waitFor.js

```diff
@@ -1,16 +1,25 @@
 // @flow
 
+import React from 'react';
```

Comment:
Can we do this?
```suggestion
import * as React from 'react';
```
It's a recommended way to import stuff from React recently